### PR TITLE
Feature/user apis

### PR DESCRIPTION
**Pull request by yuvraj1902**

### CODE REVIEW

File: controllers/index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const User = require("./user.controller");
+const Event = require("./event.controller");
+module.exports = {
+  User,
+  Event,
+};
```
Comment (dhruvildave007) [inline controllers/index.js:6]: you can skip this file by adding controller itself in the route file same as you are importing services into the controller

File: helper/send-mail.helper.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+const nodemailer = require("nodemailer");
+const { google } = require("googleapis");
+require("dotenv").config();
```
Comment (dhruvildave007) [inline helper/send-mail.helper.js:3]: name of the file should be `mail.helper.js`.

**Review by dhruvildave007 — APPROVED** [review]

File: helper/send-mail.helper.js (hunk 2 above)
Comment (yuvraj1902) [reply helper/send-mail.helper.js:3]: Sure sir I will be make this changes in next feature branch

File: controllers/index.js (hunk 1 above)
Comment (yuvraj1902) [reply controllers/index.js:6]: Is this way is not fine sir?

File: helper/send-mail.helper.js (hunk 2 above)
Comment (yuvraj1902) [reply helper/send-mail.helper.js:3]: I will make the changes in the same branch and raise another pr

**Pull request merged**